CHANGELOG.md

```diff
@@ -14,6 +14,7 @@ and this project adheres to [semantic versioning](https://semver.org/spec/v2.0.0
 ### Deprecated
 
 ### Fixed
+- Changed `cutLineage` function to make it compatible with newer PySpark version

```

Comment:
Could you change `cutLineage` reference to `cut_lineage` please.

Reply:
All done